src/components/Common/FormField.tsx

```diff
@@ -49,6 +51,7 @@ const FormField = ({
       )}
       <StyledContainer>
         <StyledErrorIBox
+          disabled={disabled}
           redErrorIcon={redErrorIcon}
           type={isType}
           placeholder={placeholder}
```

Comment:
StyledErrorIBox가 받는 상당수의 props는 키와 밸류의 네이밍이 같습니다.
이런 경우 굳이 props를 destructing하여 받지 않고 props로 받은 후 

```suggestion
          <StyledErrorIBox {...props} ...other props />
```

로 사용하여 불필요한 줄 수를 줄일 수 있습니다.